### ISSUE TITLE
Stack traces in sentry UI are incorrect when images are uploaded

### ISSUE THREAD

**Issue by Jake-Shadle** [getsentry/sentry-rust#386]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io) or self-hosted/on-premise (which version?) SaaS

Which SDK and version?
Rust 0.23.0 patched to the f8be1b86cedb5524bb1d086044680c8c920306d3 commit

### Steps to Reproduce

1. Using the patched version with Windows debug image support, report a normal panic with a backtrace

### Expected Result

The expected result is that the stack trace in the sentry UI for the event should basically be the same raw stacktrace that was reported, but with debug images, it should be slightly more accurate, as well as have the inline sources when available, as long as debug images are uploaded for the frames.

### Actual Result

Here is the raw stacktrace we send for the panic, this one is accurate.

```json
"raw_stacktrace": {
    "frames": [
        {
            "function": "RtlUserThreadStart",
            "package": "C:\\Windows\\SYSTEM32\\ntdll.dll",
            "in_app": true,
            "instruction_addr": "0x7ff81867a271"
        },
        {
            "function": "BaseThreadInitThunk",
            "package": "C:\\Windows\\System32\\KERNEL32.DLL",
            "in_app": true,
            "instruction_addr": "0x7ff816e27974"
        },
        {
            "function": "__scrt_common_main_seh",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "exe_common.inl",
            "abs_path": "D:\\agent\\_work\\9\\s\\src\\vctools\\crt\\vcstartup\\src\\startup\\exe_common.inl",
            "lineno": 288,
            "in_app": true,
            "instruction_addr": "0x7ff6ba205a8c"
        },
        {
            "function": "invoke_main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "exe_common.inl",
            "abs_path": "D:\\agent\\_work\\9\\s\\src\\vctools\\crt\\vcstartup\\src\\startup\\exe_common.inl",
            "lineno": 78,
            "in_app": true,
            "instruction_addr": "0x7ff6ba205a8c"
        },
        {
            "function": "main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "in_app": true,
            "instruction_addr": "0x7ff6b4091156"
        },
        {
            "function": "std::rt::lang_start<T>",
            "raw_function": "std::rt::lang_start<tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\rt.rs",
            "lineno": 62,
            "in_app": false,
            "instruction_addr": "0x7ff6b409102f"
        },
        {
            "function": "std::rt::lang_start_internal",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\rt.rs",
            "lineno": 45,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panic::catch_unwind",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panic.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panic.rs",
            "lineno": 434,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panicking::try",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panicking.rs",
            "lineno": 365,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panicking::try::do_call",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panicking.rs",
            "lineno": 401,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::rt::lang_start_internal::closure$2",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\rt.rs",
            "lineno": 45,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panic::catch_unwind",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panic.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panic.rs",
            "lineno": 434,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panicking::try",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panicking.rs",
            "lineno": 365,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::panicking::try::do_call",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\/library\\std\\src\\panicking.rs",
            "lineno": 401,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "core::ops::function::impls::impl$2::call_once",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "function.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\core\\src\\ops\\function.rs",
            "lineno": 259,
            "in_app": false,
            "instruction_addr": "0x7ff6b40b5546"
        },
        {
            "function": "std::rt::lang_start::closure$0<T>",
            "raw_function": "std::rt::lang_start::closure$0<tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\rt.rs",
            "lineno": 63,
            "in_app": false,
            "instruction_addr": "0x7ff6b4091061"
        },
        {
            "function": "std::sys_common::backtrace::__rust_begin_short_backtrace<T>",
            "raw_function": "std::sys_common::backtrace::__rust_begin_short_backtrace<void (*)(),tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "backtrace.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\sys_common\\backtrace.rs",
            "lineno": 125,
            "in_app": false,
            "instruction_addr": "0x7ff6b40912ab"
        },
        {
            "function": "core::ops::function::FnOnce::call_once<T>",
            "raw_function": "core::ops::function::FnOnce::call_once<void (*)(),tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "function.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\core\\src\\ops\\function.rs",
            "lineno": 227,
            "in_app": false,
            "instruction_addr": "0x7ff6b40911fb"
        },
        {
            "function": "ark_client::main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "bin.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\client\\src\\bin.rs",
            "lineno": 29,
            "in_app": true,
            "instruction_addr": "0x7ff6b4091102"
        },
        {
            "function": "ark_client::main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "lib.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\client\\src\\lib.rs",
            "lineno": 56,
            "in_app": true,
            "instruction_addr": "0x7ff6b40916b6"
        },
        {
            "function": "ark_bootstrap_client::entry::entrypoint<T>",
            "raw_function": "ark_bootstrap_client::entry::entrypoint<ark_client::main::closure$0,ark_client::main::closure$1>",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "entry.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\bootstrap-client\\src\\entry.rs",
            "lineno": 236,
            "in_app": true,
            "instruction_addr": "0x7ff6b40c99c2"
        },
        {
            "function": "ark_client::main::closure$0",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "lib.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\client\\src\\lib.rs",
            "lineno": 60,
            "in_app": true,
            "instruction_addr": "0x7ff6b42ab717"
        },
        {
            "function": "enum$<T>::run",
            "raw_function": "enum$<ark_cli::cmds::Command>::run",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "cmds.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\cli\\src\\cmds.rs",
            "lineno": 302,
            "in_app": true,
            "instruction_addr": "0x7ff6b40dac6e"
        },
        {
            "function": "ark_cli::cmds::test_crash::Command::run",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "test_crash.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\cli\\src\\cmds\\test_crash.rs",
            "lineno": 56,
            "in_app": true,
            "instruction_addr": "0x7ff6b447b4b4"
        },
        {
            "function": "std::panicking::begin_panic<T>",
            "raw_function": "std::panicking::begin_panic<str>",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\panicking.rs",
            "lineno": 541,
            "in_app": false,
            "instruction_addr": "0x7ff6b417085f"
        },
        {
            "function": "std::sys_common::backtrace::__rust_end_short_backtrace<T>",
            "raw_function": "std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic::closure$0,never$>",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "backtrace.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\sys_common\\backtrace.rs",
            "lineno": 141,
            "in_app": false,
            "instruction_addr": "0x7ff6b41707df"
        }
    ]
}
```

And here is the stacktrace after sentry has attempted to symbolicate it.

```json
"stacktrace": {
    "frames": [
        {
            "function": "RtlCheckSandboxedToken",
            "symbol": "RtlCheckSandboxedToken",
            "package": "C:\\Windows\\SYSTEM32\\ntdll.dll",
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff81867a270"
        },
        {
            "function": "GlobalReAlloc",
            "symbol": "GlobalReAlloc",
            "package": "C:\\Windows\\System32\\KERNEL32.DLL",
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff816e27973"
        },
        {
            "function": "Concurrency::details::WorkSearchContext::SearchCacheLocal",
            "symbol": "Concurrency::details::WorkSearchContext::SearchCacheLocal",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "SearchAlgorithms.cpp",
            "abs_path": "D:\\agent\\_work\\9\\s\\src\\vctools\\crt\\crtw32\\concrt\\SearchAlgorithms.cpp",
            "lineno": 1277,
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6ba205a8b"
        },
        {
            "function": "Concurrency::details::WorkSearchContext::SearchCacheLocal",
            "symbol": "Concurrency::details::WorkSearchContext::SearchCacheLocal",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "SearchAlgorithms.cpp",
            "abs_path": "D:\\agent\\_work\\9\\s\\src\\vctools\\crt\\crtw32\\concrt\\SearchAlgorithms.cpp",
            "lineno": 1277,
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6ba205a8b"
        },
        {
            "function": "main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "in_app": true,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b4091156"
        },
        {
            "function": "std::rt::lang_start<T>",
            "raw_function": "std::rt::lang_start<tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\rt.rs",
            "lineno": 62,
            "in_app": false,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b409102f"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::panicking::default_hook",
            "symbol": "std::panicking::default_hook",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "panicking.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\\\library\\std\\src\\panicking.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40b5545"
        },
        {
            "function": "std::rt::lang_start::closure$0<T>",
            "raw_function": "std::rt::lang_start::closure$0<tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "rt.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\rt.rs",
            "lineno": 63,
            "in_app": false,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b4091061"
        },
        {
            "function": "std::sys_common::backtrace::__rust_begin_short_backtrace<T>",
            "raw_function": "std::sys_common::backtrace::__rust_begin_short_backtrace<void (*)(),tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "backtrace.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\std\\src\\sys_common\\backtrace.rs",
            "lineno": 125,
            "in_app": false,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b40912ab"
        },
        {
            "function": "core::ops::function::FnOnce::call_once<T>",
            "raw_function": "core::ops::function::FnOnce::call_once<void (*)(),tuple$<> >",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "function.rs",
            "abs_path": "/rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\core\\src\\ops\\function.rs",
            "lineno": 227,
            "in_app": false,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b40911fb"
        },
        {
            "function": "ark_client::main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "bin.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\client\\src\\bin.rs",
            "lineno": 29,
            "pre_context": [
                "        // Otherwise a bunch of crates will fight over it",
                "        CoInitializeEx(std::ptr::null_mut(), COINIT_APARTMENTTHREADED)",
                "    };",
                ""
            ],
            "context_line": "    let result = ark_client::main();",
            "post_context": [
                "",
                "    #[cfg(windows)]",
                "    #[allow(unsafe_code)]",
                "    unsafe {",
                "        // Shutdown COM"
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b4091102"
        },
        {
            "function": "ark_client::main",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "lib.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\client\\src\\lib.rs",
            "lineno": 56,
            "pre_context": [
                "    #[cfg(feature = \"with_cli\")]",
                "    if let (Some(cmd), mut base_config) = (config.cmd.clone(), config.base.clone()) {",
                "        cmd.override_base_config(&mut base_config);",
                ""
            ],
            "context_line": "        return ark_bootstrap_client::entrypoint(",
            "post_context": [
                "            base_config.clone(),",
                "            &build_info,",
                "            ark_bootstrap_client::RunMode::Cli,",
                "            |params| cmd.run(base_config, params),",
                "            Box::new(ark_bootstrap_client::panic::panic_hook_stderr),"
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "missing_symbol"
            },
            "instruction_addr": "0x7ff6b40916b6"
        },
        {
            "function": "ark_bootstrap_client::entry::entrypoint<T>",
            "raw_function": "ark_bootstrap_client::entry::entrypoint<ark_client::main::closure$0,ark_client::main::closure$1>",
            "symbol": "ark_bootstrap_client::entry::entrypoint<ark_client::main::closure$0,ark_client::main::closure$1>",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "entry.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\bootstrap-client\\src\\entry.rs",
            "lineno": 161,
            "pre_context": [
                "    // create shared http client",
                "    let auth_handle = auth.as_ref().map(SignInFlow::get_handle);",
                "",
                "    let http_client = if config.offline {"
            ],
            "context_line": "        None",
            "post_context": [
                "    } else {",
                "        let auth_proxies = ark_service_framework::client::Proxy {",
                "            destination_host: \"language.ml.embark.net\",",
                "            proxy_scheme: \"https\",",
                "            proxy_host: \"public.language.ml.embark.net\","
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40c99c1"
        },
        {
            "function": "ark_module_store::crate_module_store::_::impl$0::deserialize::impl$0::visit_str<T>",
            "raw_function": "ark_module_store::crate_module_store::_::impl$0::deserialize::impl$0::visit_str<serde_json::error::Error>",
            "symbol": "ark_module_store::crate_module_store::_::impl$0::deserialize::impl$0::visit_str<serde_json::error::Error>",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "crate_module_store.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\module-store\\src\\crate_module_store.rs",
            "lineno": 34,
            "pre_context": [
                "    WasiProgram,",
                "}",
                "",
                "#[derive("
            ],
            "context_line": "    Copy, Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, EnumString, EnumVariantNames,",
            "post_context": [
                ")]",
                "#[strum(ascii_case_insensitive)]",
                "pub enum CrateBuildConfig {",
                "    Debug,",
                "    Release,"
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b42ab716"
        },
        {
            "function": "puffin::clean_function_name",
            "symbol": "puffin::clean_function_name",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "lib.rs",
            "abs_path": "C:\\Users\\ark\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\puffin-0.10.0\\src\\lib.rs",
            "lineno": 684,
            "pre_context": [
                "#[doc(hidden)]",
                "#[inline]",
                "pub fn clean_function_name(name: &str) -> &str {",
                "    if let Some(colon) = name.rfind(\"::\") {"
            ],
            "context_line": "        if let Some(colon) = name[..colon].rfind(\"::\") {",
            "post_context": [
                "            // \"foo::bar::baz::function_name\" -> \"baz::function_name\"",
                "            &name[colon + 2..]",
                "        } else {",
                "            // \"foo::function_name\" -> \"foo::function_name\"",
                "            name"
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b40dac6d"
        },
        {
            "function": "ark_cli::cmds::session::impl$3::from_arg_matches",
            "symbol": "ark_cli::cmds::session::impl$3::from_arg_matches",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "session.rs",
            "abs_path": "C:\\b\\win-3\\embark-studios\\ark\\components\\cli\\src\\cmds\\session.rs",
            "lineno": 6,
            "pre_context": [
                "use ark_error::anyhow::{self, Context as _};",
                "use ark_session_client::{self as asc, JoinSessionLink};",
                "use ark_url::{ArkLink, UrlStyle};",
                ""
            ],
            "context_line": "#[derive(Clone, Debug, clap::Subcommand, clap::ArgEnum)]",
            "post_context": [
                "pub enum Command {",
                "    /// Creates a new session",
                "    Create {",
                "        /// Make the session private. Private sessions are only joinable via link.",
                "        #[clap(long)]"
            ],
            "in_app": true,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b447b4b3"
        },
        {
            "function": "alloc::alloc::Global::grow_impl",
            "symbol": "alloc::alloc::Global::grow_impl",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "alloc.rs",
            "abs_path": "\\rustc\\c8dfcfe046a7680554bf4eb612bad840e7631c4b\\library\\alloc\\src\\alloc.rs",
            "lineno": 215,
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b417085e"
        },
        {
            "function": "alloc::alloc::Global::grow_impl",
            "symbol": "alloc::alloc::Global::grow_impl",
            "package": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
            "filename": "lib.rs",
            "abs_path": "C:\\Users\\ark\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\rfd-0.5.1\\src\\lib.rs",
            "lineno": 1,
            "context_line": "mod backend;",
            "post_context": [
                "",
                "mod file_handle;",
                "pub use file_handle::FileHandle;",
                "",
                "mod dialog;"
            ],
            "in_app": false,
            "data": {
                "symbolicator_status": "symbolicated"
            },
            "instruction_addr": "0x7ff6b41707df"
        }
    ]
}
```

This is despite having accurate debug, unwinding, and source information for the ark-client executable.

```json
{
    "code_id": "6177bb2782c7000",
    "code_file": "C:\\b\\win-3\\embark-studios\\ark\\target\\test_crash\\windows\\ark-client.exe",
    "debug_id": "604af986-9713-c10e-4c4c-44205044422e-1",
    "debug_file": "C:\\b\\win-3\\embark-studios\\ark\\target\\x86_64-pc-windows-msvc\\debug\\deps\\ark_client.pdb",
    "arch": "x86_64",
    "image_addr": "0x7ff6b4091000",
    "image_size": 102453542,
    "image_vmaddr": "0x1000",
    "candidates": [
        {
            "download": {
                "status": "notfound"
            },
            "source": "sentry:amd",
            "source_name": "AMD"
        },
        {
            "download": {
                "status": "notfound"
            },
            "source": "sentry:android",
            "source_name": "Android"
        },
        {
            "download": {
                "details": "",
                "status": "noperm"
            },
            "source": "sentry:intel",
            "source_name": "Intel"
        },
        {
            "download": {
                "details": "",
                "status": "noperm"
            },
            "source": "sentry:intel",
            "source_name": "Intel"
        },
        {
            "download": {
                "details": "",
                "status": "noperm"
            },
            "source": "sentry:intel",
            "source_name": "Intel"
        },
        {
            "download": {
                "details": "",
                "status": "noperm"
            },
            "source": "sentry:intel",
            "source_name": "Intel"
        },
        {
            "download": {
                "status": "notfound"
            },
            "source": "sentry:ios",
            "source_name": "Apple"
        },
        {
            "download": {
                "status": "notfound"
            },
            "source": "sentry:microsoft",
            "source_name": "Microsoft"
        },
        {
            "download": {
                "status": "notfound"
            },
            "source": "sentry:nvidia",
            "source_name": "NVIDIA"
        },
        {
            "download": {
                "features": {
                    "has_debug_info": false,
                    "has_sources": true,
                    "has_symbols": false,
                    "has_unwind_info": false
                },
                "status": "ok"
            },
            "location": "sentry://project_debug_file/159467887",
            "source": "sentry:project",
            "source_name": "Sentry"
        },
        {
            "download": {
                "features": {
                    "has_debug_info": false,
                    "has_sources": false,
                    "has_symbols": false,
                    "has_unwind_info": true
                },
                "status": "ok"
            },
            "location": "sentry://project_debug_file/159467888",
            "source": "sentry:project",
            "source_name": "Sentry"
        },
        {
            "debug": {
                "status": "ok"
            },
            "download": {
                "features": {
                    "has_debug_info": true,
                    "has_sources": false,
                    "has_symbols": true,
                    "has_unwind_info": false
                },
                "status": "ok"
            },
            "location": "sentry://project_debug_file/159467893",
            "source": "sentry:project",
            "source_name": "Sentry"
        }
    ],
    "debug_status": "found",
    "features": {
        "has_debug_info": true,
        "has_sources": false,
        "has_symbols": true,
        "has_unwind_info": false
    },
    "type": "symbolic"
}
```

Basically, it seems that other than the symbols from nt/kernel32, the frames that are marked "symbolicated" are incorrect, and the ones marked "missing_symbol" are...correct. So something really weird is going on there. To be be clear, this only affects stacktraces we report via a panic/error, reporting a full minidump with the same exact debug symbols is processed by sentry correctly.

As far as I can tell, this has only happened to use on Windows, we were just never uploading debug images before since that feature didn't exist in the sentry-debug-images crate, so we haven't had the incorrect symbolication problem before this. The panics we report from linux/mac that have had debug images since forever have not had this issue.
